Gemfile

```diff
@@ -1,6 +1,6 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
```

Comment:
@mogox cool, does this mean there's still a chance for this PR getting merged in? 😅